README.md

```diff
@@ -21,6 +21,6 @@
 
 ## Acknowledgements
 
-_This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org)._
- 
+_This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).
```

Comment:
Why the removing of _? I think it is the mark down syntax you removed.

Reply:
But if you delete then this markdown is essentially not usable, because there was _ infront of the text.

Reply:
Previously, I think this is the same format, and it passed CI checks

Reply:
Previously they were passing for me too, what do you want me to do here, I will try that